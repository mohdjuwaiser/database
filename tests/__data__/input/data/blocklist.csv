channel,ref
AnimalPlanetAfrica.za,https://github.com/iptv-org/iptv/issues/1831