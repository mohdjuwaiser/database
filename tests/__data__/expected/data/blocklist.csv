channel,ref
HGTVHungary.hu,https://github.com/iptv-org/iptv/issues/1831